apps/meteor/client/views/account/omnichannel/OmnichannelPreferencesPage.tsx

```diff
@@ -46,22 +46,24 @@ const OmnichannelPreferencesPage = (): ReactElement => {
 	};
 
 	return (
-		<Page>
-			<Page.Header title={t('Omnichannel')}>
-				<ButtonGroup>
-					<Button primary disabled={!isDirty} onClick={handleSubmit(handleSave)}>
-						{t('Save_changes')}
-					</Button>
-				</ButtonGroup>
-			</Page.Header>
-			<Page.ScrollableContentWithShadow>
-				<Box maxWidth='x600' w='full' alignSelf='center'>
-					<Accordion>
-						<PreferencesConversationTranscript register={register} />
-					</Accordion>
-				</Box>
-			</Page.ScrollableContentWithShadow>
-		</Page>
+		<form onSubmit={handleSubmit(handleSave)}>
+			<Page>
+				<Page.Header title={t('Omnichannel')}>
+					<ButtonGroup>
+						<Button type='submit' primary disabled={!isDirty}>
+							{t('Save_changes')}
+						</Button>
+					</ButtonGroup>
+				</Page.Header>
+				<Page.ScrollableContentWithShadow>
+					<Box maxWidth='x600' w='full' alignSelf='center'>
+						<Accordion>
+							<PreferencesConversationTranscript register={register} />
+						</Accordion>
+					</Box>
+				</Page.ScrollableContentWithShadow>
+			</Page>
+		</form>
```

Comment:
I dont think throwing the whole page inside the form is the way to go. I think setting `Page.ScrollableContentWithShadow` prop as `is='form'` would be better. Ex.:
```
<Page.ScrollableContentWithShadow is='form' onSubmit={handleSubmit(handleSave)}>
```
Unfortunately this would not work for the submit button since it's outside the form, but it will work for submitting using the `enter` key. For the button you can leave as it was:
 ```
 <Button primary disabled={!isDirty} onClick={handleSave}>
	{t('Save_changes')}
</Button>
 ```